### ISSUE TITLE
Document how to transition to Resoto

### ISSUE THREAD

**Issue by TheCatLady** [someengineering/cloudkeeper#530]
### Motivation

Existing users will need to transition to the new images.

### Desired Behavior

An announcement & docs page describing how to migrate installs to use the new images.

### Code of Conduct

- [X] I agree to follow the Code of Conduct

**Comment by aquamatthias** [comment]
@TheCatLady can we close this ticket?

**Comment by TheCatLady** [comment]
Yeah, I think we should.  We can open a new issue on the `resoto.com` repo if necessary.
